src/Microsoft.Health.ControlPlane.CosmosDb/Features/Storage/Rbac/CosmosIdentityProviderExtensions.cs

```diff
@@ -10,15 +10,15 @@ namespace Microsoft.Health.ControlPlane.CosmosDb.Features.Storage.Rbac
 {
     public static class CosmosIdentityProviderExtensions
     {
-        public static IdentityProvider ToIdentityProvider(this CosmosIdentityProvider cosmosIdentityProvider)
+        internal static IdentityProvider ToIdentityProvider(this CosmosIdentityProvider cosmosIdentityProvider)
         {
             EnsureArg.IsNotNull(cosmosIdentityProvider);
 
             return new IdentityProvider(
                 cosmosIdentityProvider.Name,
                 cosmosIdentityProvider.Authority,
                 cosmosIdentityProvider.Audience,
-                cosmosIdentityProvider.ETag.Trim('"'));
+                cosmosIdentityProvider.Version);
```

Comment:
Does version ever get set on the cosmosidentityprovider? And if it does, does it auto-increment when you send in a new version of the identity provider?